cli/command/run/cmd.go

```diff
@@ -97,6 +98,7 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 	cmd.Flags().IntVar(&concurrency, "ccy", 1, "Concurrency specifies how many suites are run at the same time.")
 	cmd.Flags().StringVar(&tunnelID, "tunnel-id", "", "Sets the sauce-connect tunnel ID to be used for the run.")
 	cmd.Flags().StringVar(&tunnelParent, "tunnel-parent", "", "Sets the sauce-connect tunnel parent to be used for the run.")
+	cmd.Flags().StringVar(&runnerVersion, "runner-version", "", "Sets runner_version.")
```

Comment:
This should be a hidden setting. We don't really want users to be using this as part of their regular workflow.

Reply:
I see. updating.